types/types.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+type Device struct {
+	Title          string
+	Description    string
+	Location       string
+	FeaturedSensor *Sensor
+	Tags           []string
```

Comment:
Create type Tag instead of string[] and save tags to db as standalone document? Better for tag cloud and searching. Or not?